python/ray/worker.py

```diff
@@ -1296,6 +1161,7 @@ def init(redis_address=None,
          raylet_socket_name=None,
          temp_dir=None,
          load_code_from_local=False,
+         enable_profiling=True,
```

Comment:
I still prefer to leave this out. If you *really* want to keep it, then I suggest we take it out of `ray.init()` but leave it in the internal implementation.

Reply:
I'm removing it from this PR and created an issue: https://github.com/ray-project/ray/issues/4618